config/templates/stack-template.json

```diff
@@ -65,7 +64,7 @@
       {{if .Experimental.WaitSignal.Enabled}}
       "CreationPolicy" : {
         "ResourceSignal" : {
-          "Count" : "{{.WorkerCount}}",
+          "Count" : "{{.MinWorkerCount}}",
```

Comment:
Sure, if that's appropriate. Since this is the `CreationPolicy` I thought it didn't apply to stack updates at all?